common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1590,6 +1590,18 @@ public static String computeMaxHostBrokerProtocol() {
         public static final String AUTHORITY =
                 "content://com.microsoft.intune.shareduserlessdataclear/datacollection";
 
+        /**
+         * Intune CompanyPortal's ContentProvider String Authority constant for Clearing App policies.
+         */
+        public static final String AUTHORITY_COMPANY_PORTAL =
```

Comment:
COMPANY_PORTAL_CONTENT_PROVIDER_AUTHORITY

Reply:
nit: I think these two constants should go into separate class.. CompanyPortalContentProviderCall